hydragnn/utils/abstractrawdataset.py

```diff
@@ -120,6 +120,12 @@ def __init__(self, config, dist=False, sampling=None):
                     "PointPairFeatures"
                 ]
 
+        if self.spherical_coordinates:
+            self.spherical_coordinates_transform = Spherical(norm=False)
+
+        if self.point_pair_features:
+            self.point_pair_features_transform = PointPairFeatures()
```

Comment:
Wondering if something like
```suggestion
        if self.spherical_coordinates:
            self.edge_feature_transform = Spherical(norm=False, cat=False)

        if self.point_pair_features:
            self.edge_feature_transform = PointPairFeatures(cat=False)
```
is simpler, then check if `hasattr()` in line 263, and update/merge lines 389-403.